src/app/mytools/my-tool/my-tool.component.scss

```diff
@@ -0,0 +1,15 @@
+/*
```

Comment:
Is this intentional? This looks like all copyright?

Reply:
Yes, Angular CLI auto-generates an SCSS file for each component and hooks the .ts file to it.  We've mostly kept the empty scss file in case it's needed.  My new copyright add-on automatically adds the copyright to new files.